requirements/base.txt

```diff
@@ -16,3 +16,8 @@ requests==2.12.3          # via slumber
 six==1.10.0               # via django-extensions
 slumber==0.7.1            # via edx-rest-api-client
 unicodecsv==0.14.1
+djangorestframework==3.5.3
+django-filter==1.0.1
+edx-django-oauth2-provider==1.1.4
+djangorestframework-oauth==1.1.0
+edx-drf-extensions==1.2.2
```

Comment:
@saleem-latif @asadiqbal08 @zubair-arbi @haikuginger @bradenmacdonald Please don't modify requirements files directly and don't approve/merge PRs that contain such changes. Spent ~ 20 minutes trying to figure out why my new tests does not run properly only to find out that `make upgrade && make requirements` removes a couple of required packages and find out which ones.

In particular, `edx-django-oauth2-provider`, `djangorestframework-oauth` and `edx-drf-extensions` was misisng, since they are not listed in the `base.in`